Makefile

```
@@ -1,15 +1,9 @@
-.PHONY: quality style test
+.PHONY: test commit
 
 check_dirs := examples tests trl
 
 test:
 	python -m pytest -n auto --dist=loadfile -s -v ./tests/
 
-quality:
-	black --check --line-length 119 --target-version py38 $(check_dirs)
-	isort --check-only $(check_dirs)
-	flake8 $(check_dirs)
-
-style:
-	black --line-length 119 --target-version py38 $(check_dirs)
-	isort $(check_dirs)
\ No newline at end of file
+commit:
```

small nit, let's rename it otherwise it might sound like the actually commits something :) 

```suggestion
precommit:
```

---

Done!